test/helpers/npm-registry.js

```diff
@@ -0,0 +1,68 @@
+import Docker from 'dockerode';
+import getStream from 'get-stream';
+import got from 'got';
+import pRetry from 'p-retry';
+
+const SERVER_PORT = 5984;
```

Comment:
can we change the port? It’s the CouchDB default port, if folks have running it locally, the tests will fail with `Error starting userland proxy: Bind for 0.0.0.0:5984 failed: port is already allocated`

Reply:
I made the change.